### PR TITLE
added two csvs and an expectation suite

### PR DESCRIPTION
**Pull request by talagluck**
Added two csvs to the data folder and the `npi.demo` expectation suite. `good.csv` will successfully pass; `really_bad.csv` will partially fail. These will need to be moved to an S3 bucket so we can show how to validate on non-local data (see [here](https://github.com/superconductive/great_expectations_action/issues/18).

### CODE REVIEW

**Comment by hamelsmu** [comment]
@talagluck what are the name of the checkpoints I need to run in order to see the new test suites on the csvs in this PR?  Just to confirm the name of the checkpoint is `npi.demo`

from this branch, I ran

```bash
 great_expectations_action % great_expectations checkpoint run npi.demo
Heads up! This feature is Experimental. It may change. Please give us your feedback!
Could not find checkpoint `npi.demo`. Try running:
  - `great_expectations checkpoint list` to verify your checkpoint exists
  - `great_expectations checkpoint new` to configure a new checkpoint
```

sorry for the noob questions I am still learning GE

**Comment by Aylr** [comment]
It looks like this PR doesn't have new checkpoints - just a suite - I'll push a quick commit that adds checkpoints to this.

**Comment by hamelsmu** [comment]
I didn’t know it was possible to create a suite without a checkpoint?  I thought GE materializes both for you automatically when you start?

Just curious as I’m still learning

**Comment by Aylr** [comment]
Ah, excellent question!
- An `ExpectationSuite` is the series of assertions. While it is created in the context of data, on it's own it has no built in relationship to data.
- A `Checkpoint` is a conveninece that pairs at least one batch of data with at least one `ExpectationSuite` and runs a `ValidationOperator` which produces a `ValidationResult`

You can try making a new checkpoint with the `great_expectations checkpoint new` command. Checkpoints are materialized as `yml` files in the `great_expectations/checkpoints` directory.

**Comment by Aylr** [comment]
Here's all I did to create two new checkpoints:

```bash
❯ great_expectations suite list
3 Expectation Suites found:
 - npi.demo
 - suite_will_fail
 - suite_will_pass
 ~/repos/great_expectations_action  demo_data_and_suite *1 ?1         21s  .venv 🐍 
❯ great_expectations checkpoint new
Usage: great_expectations checkpoint new [OPTIONS] CHECKPOINT SUITE
Try 'great_expectations checkpoint new --help' for help.

Error: Missing argument 'CHECKPOINT'.
 ~/repos/great_expectations_action  demo_data_and_suite *1 ?1               .venv 🐍 
❯ great_expectations checkpoint new npi.pass npi.demo
Heads up! This feature is Experimental. It may change. Please give us your feedback!

Would you like to:
    1. choose from a list of data assets in this datasource
    2. enter the path of a data file
: 1

Which data would you like to use?
    1. good (file)
    2. npi_tiny (file)
    3. really_bad (file)
    Don't see the name of the data asset in the list above? Just type it
: 1
A checkpoint named `npi.pass` was added to your project!
  - To edit this checkpoint edit the checkpoint file: /Users/taylor/repos/great_expectations_action/great_expectations/checkpoints/npi.pass.yml
  - To run this checkpoint run `great_expectations checkpoint run npi.pass`
 ~/repos/great_expectations_action  demo_data_and_suite *1 ?2          7s  .venv 🐍 
❯ great_expectations checkpoint new npi.fail npi.demo
Heads up! This feature is Experimental. It may change. Please give us your feedback!

Would you like to:
    1. choose from a list of data assets in this datasource
    2. enter the path of a data file
: 1

Which data would you like to use?
    1. good (file)
    2. npi_tiny (file)
    3. really_bad (file)
    Don't see the name of the data asset in the list above? Just type it
: 3
A checkpoint named `npi.fail` was added to your project!
  - To edit this checkpoint edit the checkpoint file: /Users/taylor/repos/great_expectations_action/great_expectations/checkpoints/npi.fail.yml
  - To run this checkpoint run `great_expectations checkpoint run npi.fail`
 ~/repos/great_expectations_action  demo_data_and_suite *1 ?3          4s  .venv 🐍 
❯ 
```

**Comment by hamelsmu** [comment]
Hmmm something is wrong somehow the action picked up the npi checkpoint and ignored the other old checkpoints even though these new npi ones are not referenced in the action yet 🤔

**Comment by Aylr** [comment]
Oooh this is looking really cool

File: great_expectations/expectations/npi/demo.json [hunk 1]
```diff
@@ -0,0 +1,633 @@
...
+      },
+      "meta": {}
+    }
+  ],
+  "meta": {
+    "BasicSuiteBuilderProfiler": {
+      "batch_kwargs": {
+        "data_asset_name": "npi",
+        "datasource": "csvs",
+        "path": "/Users/talgluck/Documents/crushing-pipeline-debt-demo/great_expectations/../data/npi.csv"
```
Comment (hamelsmu) [inline great_expectations/expectations/npi/demo.json:389]: @Aylr @talagluck does this path need to be corrected?

File: great_expectations/expectations/npi/demo.json [hunk 2]
```diff
@@ -0,0 +1,633 @@
...
+      },
+      "created_at": 1595943301.317559,
+      "created_by": "BasicSuiteBuilderProfiler"
+    },
+    "citations": [
+      {
+        "batch_kwargs": {
+          "data_asset_name": "npi",
+          "datasource": "data__dir",
+          "path": "/Users/talgluck/Documents/crushing-pipeline-debt-demo/great_expectations/../data/npi.csv"
```
Comment (hamelsmu) [inline great_expectations/expectations/npi/demo.json:399]: Same question is this path correct?

**Comment by hamelsmu** [comment]
I corrected the path myself no worries

**Pull request merged**